pyproject.toml

```diff
@@ -190,6 +186,7 @@ lint.ignore = [
   "PT017",  # pytest-assert-in-except
   "PYI026", # flake8-pyi: typing.TypeAlias added in Python 3.10
   "PYI034", # flake8-pyi: typing.Self added in Python 3.11
+  "UP038",  # pyupgrade: deprecated rule
```

Comment:
Could you talk about why we're adding a deprecated rule?

Reply:
The opposite, ignoring a deprecated rule :)

Reply:
Oh, right!